components/services/getAddress/__tests__/getAddress.test.js

```diff
@@ -0,0 +1,35 @@
+/* globals describe,test,expect */
```

Comment:
Really good that you started writing unit tests on IUI. +1
Please add these globals into `package.json` globals if possible.

Reply:
this comment is to prevent standard error on not knowing these jest related methods, do you have any sample on resolving this with a better approach?